tests/DiContainer/Exceptions/ExceptionsTest.php

```diff
@@ -20,6 +20,7 @@
 use function Kaspi\DiContainer\diAutowire;
 
 /**
+ * @covers \Helper::functionName
```

Comment:
при запуске тестов с покрытием вываливает предупреждение:
```txt
"@covers \Helper::functionName" is invalid
```

```suggestion
 * @covers \Kaspi\DiContainer\Helper
```